### PR TITLE
fix(route): 纽约时报 - 新闻简报 图片缺失

### LINKED ISSUE #7835: 纽约时报每日简报存在多张图片时只能显示第一张
#### 路由地址（不包含参数）

```
nytimes/daily_briefing_chinese 
```

#### 完整路由地址，包含所有必选与可选参数

```
https://rsshub.rssforever.com/nytimes/daily_briefing_chinese 
```

#### 相关文档地址

- https://docs.rsshub.app/traditional-media.html#niu-yue-shi-bao

#### 预期是什么
图片正常显示

#### 实际发生了什么
每日简报存在多张图片时只能显示第一张

#### 部署相关信息

| Env                | Value         |
| ------------------ | ------------- |
| OS                 |               |
| Node version       |               |
| if Docker, version |               |

- 额外信息（日志、报错等）

### PR DESCRIPTION
**Pull request by auto-bot-ty**
## 该 PR 相关 Issue / Involved issue

Close #7835

## 完整路由地址 / Example for the proposed route(s)

```routes
/nytimes/daily_briefing_chinese
```

## 新RSS检查列表 / New RSS Script Checklist

- [ ] 这个PR中包含了新的路由吗? Does this PR add new route?
  - 如果有, 请完成检查列表. If yes, please finish the check list
  - **如果你的PR符合下方某个事项, 也请注明. If any of the checklist item meets your PR, please fill it out.**
  - [x] <- 这样打勾
- [ ] 是否提供了文档? Documentation provided?
  - [ ] 是否提供了英文文档? EN Documentation provided?
- [ ] 是否支持全文获取? Is this RSS Script support fulltext?
  - [ ] 如果全文获取中需要访问文章链接, 是否使用了缓存? If fulltext requires to fetch detail pages, is cache used in the process?
  - [缓存说明](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun) | [How to use cache](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun)
- [ ] 目标是否有明显的反爬/频率限制? Is there any sign of anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? (延长缓存时间, 写文档说明, etc.) If yes, do your code reflect this sign? (e.g. write documentations, use long cache time)
- [ ] 目标是否有提供日期？ Is there a date in the source?
  - [ ] 如果有，包是否正确解析？ If there is, can this script provide this info?
  - [ ] 如果有提供解析能力，时区是否正确调整？ Is the timezone correctly provided?
  - 如果有提供日期，但是没有提供解析，请说明原因 If there is a date but this script does not parse, please provide your reason.
- [ ] 是否引入的新的包? Any new package introduced?
  - 如果有, 请说明原因. If yes, please state your reason
- [ ] 是否使用了`Puppeteer`? Make use of `Puppeteer`?
  - 如果有, 请说明原因. If yes, please state your reason

### CODE REVIEW

File: lib/routes/nytimes/daily_briefing_chinese.js [hunk 1]
```diff
@@ -28,15 +28,19 @@ module.exports = async (ctx) => {
...
-                        item.replaceWith(`<img src="${link}">`);
+                    $('.css-1xdhyk6.erfvjey0').each((i, e) => {
+                        const link = imageLink[i];
+                        $(e).replaceWith(`<img src="${link}">`);
                     });
-                    item.description = $('.meteredContent.css-1r7ky0e').html();
                     const date = $('.css-x7rtpa.e16638kd0').attr('datetime');
+
+                    item.description = $('.meteredContent.css-1r7ky0e').html();
                     item.pubDate = parseDate(date);
```
Comment (SettingDust) [inline lib/routes/nytimes/daily_briefing_chinese.js:44]: 日期好像不太对 
https://www.nytimes.com/zh-hans/2021/06/29/world/asia/china-hong-kong-security-law-anniversary.html

**Review by SettingDust — CHANGES_REQUESTED** [review]

File: lib/routes/nytimes/daily_briefing_chinese.js (hunk 1 above)
Comment (auto-bot-ty) [reply lib/routes/nytimes/daily_briefing_chinese.js:44]: @SettingDust 已修改

**Review by SettingDust — APPROVED** [review]
```
<span aria-hidden="true" class="css-16f3y1r e13ogyst0">Didi started trading last week on the New York Stock Exchange.</span><span class="css-cnj6d5 e1z0qqy90"><span class="css-1ly73wi e1tej78p0">Credit...</span><span>Brendan Mcdermid/Reuters</span>
```
@


图片题注可以改成center small 之类的，只是建议，已经可以合并了

**Comment by auto-bot-ty** [comment]
> ```
> <span aria-hidden="true" class="css-16f3y1r e13ogyst0">Didi started trading last week on the New York Stock Exchange.</span><span class="css-cnj6d5 e1z0qqy90"><span class="css-1ly73wi e1tej78p0">Credit...</span><span>Brendan Mcdermid/Reuters</span>
> ```
> 
> @
> 
> 图片题注可以改成center small 之类的，只是建议，已经可以合并了

是这样吗?
```
<figure class="css-jcw7oy e1g7ppur0" aria-label="media" role="group"><img src="https://static01.nyt.com/images/2021/07/12/business/CN-10china-tech-print-copy/merlin_190161306_f3d0718f-d525-4d72-8b7b-f74d67cc5c2c-articleLarge.jpg" referrerpolicy="no-referrer"><p><small>Didi started trading last week on the New York Stock Exchange.Credit...Brendan Mcdermid/Reuters</small></p></figure>
```
<figure class="css-jcw7oy e1g7ppur0" aria-label="media" role="group"><p><small>Didi started trading last week on the New York Stock Exchange.Credit...Brendan Mcdermid/Reuters</small></p></figure>

**Comment by SettingDust** [comment]
嗯，不然题注和正文分辨不便

**Comment by auto-bot-ty** [comment]
> 嗯，不然题注和正文分辨不便

已修改

**Review by NeverBehave — APPROVED** [review]

**Pull request merged**